stdlib/builtin/complex.rbs

```diff
@@ -114,77 +110,77 @@ class Complex < Numeric
   # If `cmp`'s imaginary part is zero, and `object` is also a real number (or a
   # Complex number where the imaginary part is zero), compare the real part of
   # `cmp` to object.  Otherwise, return nil.
-  # 
+  #
   #     Complex(2, 3)  <=> Complex(2, 3)   #=> nil
   #     Complex(2, 3)  <=> 1               #=> nil
   #     Complex(2)     <=> 1               #=> 1
   #     Complex(2)     <=> 2               #=> 0
   #     Complex(2)     <=> 3               #=> -1
-  # 
+  #
   def <=>: (Numeric) -> Integer?
 
   # Returns true if cmp equals object numerically.
-  # 
+  #
   #     Complex(2, 3)  == Complex(2, 3)   #=> true
   #     Complex(5)     == 5               #=> true
   #     Complex(0)     == 0.0             #=> true
   #     Complex('1/3') == 0.33            #=> false
   #     Complex('1/2') == '1/2'           #=> false
-  # 
-  def ==: (untyped) -> bool
+  #
+  def ==: (any) -> bool
 
   def >: (Numeric) -> bot
 
   def >=: (Numeric) -> bot
 
   # Returns the absolute part of its polar form.
-  # 
+  #
   #     Complex(-1).abs         #=> 1
   #     Complex(3.0, -4.0).abs  #=> 5.0
-  # 
+  #
   def abs: () -> Numeric
 
   # Returns square of the absolute value.
-  # 
+  #
   #     Complex(-1).abs2         #=> 1
   #     Complex(3.0, -4.0).abs2  #=> 25.0
-  # 
+  #
   def abs2: () -> Numeric
 
   # Returns the angle part of its polar form.
-  # 
+  #
   #     Complex.polar(3, Math::PI/2).arg  #=> 1.5707963267948966
-  # 
+  #
   def angle: () -> Float
 
   # Returns the angle part of its polar form.
-  # 
+  #
   #     Complex.polar(3, Math::PI/2).arg  #=> 1.5707963267948966
-  # 
+  #
   alias arg angle
 
   def ceil: (*untyped) -> bot
 
-  def clone: (?freeze: bool) -> self
+  def clone: -> self
 
-  def coerce: (Numeric) -> [ Complex, Complex ]
+  def coerce: (Numeric) -> [ Complex, self ]
```

Comment:
I'm not sure `self` is better here. Could you explain the background of the changes from `Complex` to `self`?